src/components/Button/index.ts

```diff
@@ -1 +1 @@
-export { Button } from './Button'
+export { Button,buttonBaseStyle } from './Button'
```

Comment:
Don't export this from the index, just from the file. 
So it remains internal to the library.